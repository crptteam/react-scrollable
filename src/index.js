import Scrollable from "./components/Scrollable";

export { Scrollable };

export default Scrollable;