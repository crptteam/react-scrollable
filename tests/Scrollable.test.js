import React from "react";
import { Scrollable } from "../src/";
import { mount } from "enzyme";

describe("Scrollable", () => {
  it("Should renders without problems", () => {
    const wrapper = mount(
      <Scrollable>
        Кажется, эти придурки вырубились.
      </Scrollable>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
