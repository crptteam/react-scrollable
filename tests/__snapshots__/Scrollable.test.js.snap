// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scrollable Should renders without problems 1`] = `
<Scrollable
  theme={Object {}}
>
  <ScrollableWrap>
    <div
      className="ScrollableWrap-gosbsp gwtdCs"
    >
      <Scrollable__InnerWrap
        innerRef={[Function]}
        theme={Object {}}
      >
        <div
          className="Scrollable__InnerWrap-hefNkY cehacQ"
        >
          Кажется, эти придурки вырубились.
        </div>
      </Scrollable__InnerWrap>
      <Scrollable__Track
        innerRef={[Function]}
      >
        <div
          className="Scrollable__Track-bdedxF joqAN"
        >
          <Scrollable__Thumb
            innerRef={[Function]}
            top={0}
          >
            <div
              className="Scrollable__Thumb-lvMdc ctqOaj"
            />
          </Scrollable__Thumb>
        </div>
      </Scrollable__Track>
    </div>
  </ScrollableWrap>
</Scrollable>
`;
